tests/controllers/hibp.test.js

```diff
@@ -9,6 +9,19 @@ const { testBreaches } = require("../test-breaches");
 require("../resetDB");
 
 
+test("notify POST without token should throw error", async() => {
+  const testEmail = "victim@spoofattack.com";
+  const testHash = sha1(testEmail);
+  const testPrefix = testHash.slice(0, 6).toUpperCase();
+  const testSuffix = testHash.slice(6).toUpperCase();
+
+  const mockRequest = { body: { hashPrefix: testPrefix, hashSuffixes: [testSuffix], breachName: "SomeSensitiveBreach" } };
+  const mockResponse = { status: jest.fn(), json: jest.fn() };
+
+  await expect(hibp.notify(mockRequest, mockResponse)).rejects.toThrow("HIBP notify endpoint requires valid authorization token.");
+});
```

Comment:
Do we need another test for an invalid token throwing an error?

Reply:
Good call. Added.